src/Engine/ProtoCore/Lang/BuiltInMethods.cs

```diff
@@ -795,7 +830,8 @@ public BuiltInMethods(Core core)
                     {
                         new KeyValuePair<string, ProtoCore.Type>("object", TypeSystem.BuildPrimitiveTypeObject(PrimitiveType.kTypeVar, 0)),
                     },
-                    ID = BuiltInMethods.MethodID.kGetType
+                    ID = BuiltInMethods.MethodID.kGetType,
+                    MethodAttributes = new MethodAttributes(){Description = Resources.gettypes}
```

Comment:
Follow the same style everywhere... "gettypes" starts with lower case, whereas "ImportFileByGivenFilePathWithMode" starts with upper case, I'll prefer they start with upper case everywhere.